lib/resources/dstu2/soarian_procedure.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: Procedure
+field_name_base_url: http://hl7.org/fhir/DSTU2/procedure-definitions.html#Procedure
+fields:
+- name: code
+  type: CodeableConcept
+  binding:
+    description: A code to identify a specific procedure
+    terminology:
+    - display: Procedure Codes (SNOMED CT)
```

Comment:
As mentioned updated the binding notes as 'A code to identify a specific procedure. Only one terminology systems will be returned, depending on configuration'